services/navigation/builtin/builtin_test.go

```diff
@@ -1697,9 +1721,9 @@ func TestValidateGeometry(t *testing.T) {
 	t.Run("fail case", func(t *testing.T) {
 		cfg = createBox(r3.Vector{X: 10, Y: 10, Z: 10})
 		_, err := cfg.Validate("")
-		expectedErr := "geometries specified through the navigation are not allowed to have a translation"
 		test.That(t, err, test.ShouldNotBeNil)
-		test.That(t, err.Error(), test.ShouldEqual, expectedErr)
+		fmt.Println("err: ", err)
+		test.That(t, strings.Contains(err.Error(), errGeomWithTranslation.Error()), test.ShouldBeTrue)
```

Comment:
try `test.ShouldContainSubString`, or better yet, make them two different errors

Reply:
> make them two different errors

not sure what you mean by this. A unique error for bounding regions and obstacles?
I am not sure how I feel about this since they would share all words expect one.